cli/pcluster/config_sanity.py

```diff
@@ -423,6 +482,35 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
                     "Invalid num_of_raid_volumes. "
                     "Needs min of 2 volumes for RAID and max of 5 EBS volumes are currently supported.",
                 )
+        # FSX FS Id check
+        elif resource_type == "fsx_fs_id":
+            try:
+                ec2 = boto3.client(
+                    "ec2",
+                    region_name=self.region,
+                    aws_access_key_id=self.aws_access_key_id,
+                    aws_secret_access_key=self.aws_secret_access_key,
+                )
+                fsx = boto3.client(
+                    "fsx",
+                    region_name=self.region,
+                    aws_access_key_id=self.aws_access_key_id,
+                    aws_secret_access_key=self.aws_secret_access_key,
+                )
+                self.__check_fsx_fs_id(ec2, fsx, resource_value)
+            except ClientError as e:
+                self.__fail(resource_type, e.response.get("Error").get("Message"))
+        # FSX capacity size check
+        elif resource_type == "FSx_storage_capacity":
+            if int(resource_value) % 3600 != 0 or int(resource_value) < 3600:
```

Comment:
minor: it is enough to check that the number is positive and multiple of 3600.

`int(resource_value) % 3600 != 0 or int(resource_value) < 0`

Reply:
It's all we can do, max is not documented and min is 1. See [StorageCapacity](https://docs.aws.amazon.com/fsx/latest/APIReference/API_CreateFileSystem.html#FSx-CreateFileSystem-request-StorageCapacity)

Reply:
yeah I was simply referring to the fact that instead of having ` int(resource_value) < 3600` we can have `int(resource_value) < 0`

Reply:
changed